### DISCUSSION TITLE
Issue with sending profile configuration using a specific domain

### DISCUSSION THREAD

**Discussion by tamirhaddad** [gophish/gophish#3251] [General]
Hello,

I’m encountering an issue while configuring the sending profile in Gophish. I’m using a specific domain for my tests: [https://www.íbou.fr](https://www.xn--bou-qma.fr/), which translates to [www.xn--bou-qma.fr](http://www.xn--bou-qma.fr/) (Punycode conversion). My hosting provider is Gandi, and I successfully set up the IMAP server using the email address hello@xn--bou-qma.fr.

However, when configuring the SMTP sending profile, I’m facing issues. I tested with a standard domain from the same hosting provider, and it works fine. But with this particular domain, the mail sending test fails with the following error:

"Max connection attempts exceeded - 535 5.7.8 Error: authentication failed"

Could you help me figure out what might be causing this issue with this specific domain setup in Gophish and how to resolve it?

Thank you in advance for your assistance.

Best regards,

**Comment by g4xyk00** [comment]
Hi, can you send a screenshot of the settings? Maybe try resetting your email password and sending a test email to see if that helps.
